docs/literate/src/files/adding_new_scalar_equations.jl

```diff
@@ -41,10 +41,10 @@ equation = CubicEquation()
 initial_condition_sine(x, t, equation::CubicEquation) = SVector(sinpi(x[1]))
 
 mesh = TreeMesh(-1.0, 1.0, # min/max coordinates
-                initial_refinement_level=4,
-                n_cells_max=10^4)
+                initial_refinement_level = 4,
+                n_cells_max = 10^4)
 
-solver = DGSEM(3 #= polynomial degree =#, flux_central)
+solver = DGSEM(3, flux_central) #= polynomial degree =#
```

Comment:
```suggestion
solver = DGSEM(3, flux_central)
```
This doesn't make so much sense anymore. Should we just remove this comment?